test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -2686,6 +2747,10 @@ static int traffic_mngr_check_shaper(void)
 		return ODP_TEST_INACTIVE;
 	}
 
+	/* This test needs 1 Mbps, 4Mbps, 10 Mpbs, 40Mbps, 100 Mbps */
+	if ((tm_shaper_min_rate > MBPS * 1) || (tm_shaper_max_rate < 100 * MBPS))
```

Comment:
I do not think it is good to disable the whole test if only one of the rates falls outside the supported range. The shaper test could just run the subtests that use a supported rate and the whole test could be disabled only if none of the rates are supported.

Space is missing from 4 Mbps and 40 Mbps.

Reply:
Space is still missing from 4 Mbps.

Reply:
Sorry, somehow I missed that.